package.json

```diff
@@ -112,16 +113,22 @@
     "@fortawesome/fontawesome-svg-core": "~1.2.25",
     "@fortawesome/free-solid-svg-icons": "~5.11.2",
     "@fortawesome/react-fontawesome": "~0.1.4",
-    "chart.js": "^2.8.0",
-    "formik": "^2.0.4",
-    "moment": "^2.24.0",
-    "react-spinners": "^0.6.1",
-    "react-toastify": "^5.3.2"
+    "@tinymce/tinymce-react": "~3.3.2",
+    "chart.js": "~2.8.0",
+    "formik": "~2.0.4",
+    "moment": "~2.24.0",
+    "react-spinners": "~0.6.1",
+    "react-toastify": "~5.3.2",
+    "tinymce": "~5.0.16"
   },
```

Comment:
Are all of these new dependencies needed?

(Guessing that this is a result of some merges)

Reply:
This one is a result of some merges and the version testing that @tehKapa was doing earlier. These seem to be version updates and also use `~` rather than `^`. 

Would you like me to revert them back to what they were in my initial commit?

Reply:
These are all dependencies we are actively using. We can leave them there. What do you think @jackcmeyer?